DesignSafe-CI/portal
# Task/des 2107 publication download text update

## Overview: ##
We don't allow downloads for datasets larger than 2GB. This PR changes the wording and leads the user to create a TUP account to download large files. Also removes the download button is the download is greater than 2GB, getting rid of an extra popup screen. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2107](https://tacc-main.atlassian.net/browse/DES-2107)

## Summary of Changes: ##
Used conditional statement to change text that appears in a download is greater than 2 GB. 
Used conditional statement to  determine if 'Download' button appears. 

## Testing Steps: ##
1. Find a project with a dataset size greater than 2GB. Notice the text and lack of Download button. 
2. Find a proejct with a dataset less than or equal to 2GB. The text and Download button should appear as normal.

## UI Photos:
Dataset > 2GB:
<img width="412" alt="Screen Shot 2024-01-24 at 11 44 42 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/1d79bed7-8817-436a-8acc-ea7f7348fb01">

Dataset <= 2GB:
<img width="411" alt="Screen Shot 2024-01-24 at 11 45 05 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/4beba52b-fd77-4d86-b6a0-b2839ae1b704">

## Notes: ##
